test/e2e/snaps/test-snap-bip-44.spec.js

```diff
@@ -31,11 +31,16 @@ describe('Test Snap bip-44', function () {
 
         // navigate to test snaps page and connect
         await driver.driver.get(TEST_SNAPS_WEBSITE_URL);
-        await driver.fill('.snapId3', 'npm:@metamask/test-snap-bip44');
-        await driver.clickElement({
-          text: 'Connect BIP-44 Snap',
-          tag: 'button',
-        });
+        await driver.delay(1000);
+        await driver.fill('#snapId3', 'npm:@metamask/test-snap-bip44');
+
+        // find and scroll to snapId3
+        const snapButton = await driver.findElement('#snapId4');
```

Comment:
This scrolls to `snapId4`, but we are currently working on `snapId3`, is this intentional? If so, we may need a better clarifying comment.